### DISCUSSION TITLE
Does hello os need everything only based on qt.

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#345] [General]
When I put forward some suggestions, I always have a question, but the answer is that only qt can accept it, others are ignored, but there is no way to solve the problem of "easy to use" (all the interface can be solved by the mouse)  , or when the activity of the software is dead but still sticking to use (eg KDE browser , slim DM), can't we fall back and use non-qt software?

**Comment by probonopd** [comment]
I don't understand the question.

Qt is the native toolkit used in helloSystem.
But helloSystem can also run applications that are not based on Qt, such as Firefox.
